pkg/client/unversioned/request.go

```diff
@@ -48,6 +48,16 @@ import (
 // are therefore not allowed to set manually.
 var specialParams = sets.NewString("timeout")
 
+// backoff is the underlying backoff struct we use to prevent thundering herd drama, esp when this runs in a controller.
+var backoff = &URLBackoff{Backoff: util.NewBackOff(1*time.Second, 120*time.Second)}
```

Comment:
i see in slack there was a comment about a "single client backoff without global visibility" which there seems to be consensus around.  However, clients have a throttle already.  So, that is why we originally mentioned that throttle might want to get involved...   most likely im glancing over a detail here.